app/(main)/login/page.test.tsx

```diff
@@ -87,3 +103,30 @@ describe('Login', () => {
     expect(mockPush).toHaveBeenCalledWith('/league/all');
   });
 });
+
+describe('Login loading spinner', () => {
+  it('should show the loading spinner', async () => {
+    (useStateMock as jest.Mock).mockImplementation((init: boolean) => [
+      true,
+      setIsLoading,
+    ]);
```

Comment:
Your tests seem to be failing, please look into this and make sure it's changing the loading state properly

Reply:
@choir27 I was aware :P Just been busy working on other things and unable to get back to this. TY!

Reply:
@choir27 fixed